nwb_conversion_tools/basedatainterface.py

```diff
@@ -8,6 +8,10 @@
 
 class BaseDataInterface:
 
+    @staticmethod
+    def get_ecephys_metadata(data_file_path: str):
+        raise NotImplementedError("The get_ecephys_metadata method has not been configured for this class!")
```

Comment:
In generalizing the ability of Sorting extractors and LFP extractors to write ecephys metadata in the absence of an actual Recording extractor, I believe it would be useful to just have this as a static method for each specific Recording extractor, whose own `get_metadata()` will simply wrap to it.